src/aat/java/uk/gov/hmcts/ccd/fta/steps/BackEndFunctionalTestScenarioPlayer.java

```diff
@@ -0,0 +1,270 @@
+package uk.gov.hmcts.ccd.fta.steps;
+
+import org.junit.Assert;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.http.HttpStatus;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+import java.io.IOException;
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import feign.FeignException;
+import io.cucumber.java.Before;
+import io.cucumber.java.Scenario;
+import io.cucumber.java.en.Given;
+import io.cucumber.java.en.Then;
+import io.cucumber.java.en.When;
+import io.restassured.RestAssured;
+import io.restassured.response.Response;
+import io.restassured.specification.QueryableRequestSpecification;
+import io.restassured.specification.RequestSpecification;
+import io.restassured.specification.SpecificationQuerier;
+import uk.gov.hmcts.ccd.datastore.tests.AATHelper;
+import uk.gov.hmcts.ccd.datastore.tests.helper.idam.AuthenticatedUser;
+import uk.gov.hmcts.ccd.fta.data.RequestData;
+import uk.gov.hmcts.ccd.fta.data.ResponseData;
+import uk.gov.hmcts.ccd.fta.data.UserData;
+import uk.gov.hmcts.ccd.fta.util.JsonUtils;
+
+@SuppressWarnings({"LocalVariableName"})
+public class BackEndFunctionalTestScenarioPlayer implements BackEndFunctionalTestAutomationDSL {
+
+    private static final String DYNAMIC_CONTENT_PLACEHOLDER = "[[DYNAMIC]]";
+
+    private final BackEndFunctionalTestScenarioContext scenarioContext;
+    private final AATHelper aat;
+    private Scenario scenario;
+
+    private Logger logger = LoggerFactory.getLogger(BackEndFunctionalTestScenarioPlayer.class);
+
+    public BackEndFunctionalTestScenarioPlayer() {
+        aat = AATHelper.INSTANCE;
+        RestAssured.baseURI = aat.getTestUrl();
+        RestAssured.useRelaxedHTTPSValidation();
+        scenarioContext = new BackEndFunctionalTestScenarioContext();
+    }
+
+    @Before()
+    public void prepare(Scenario scenario) {
+        this.scenario = scenario;
+    }
+
+    @Override
+    @Given("an appropriate test context as detailed in the test data source")
```

Comment:
We could move all this code and logic from this step into the @before `prepare` function since this step is here purely for data setup purposes internal to the test and will have to be the first step in every scenario we write. Also, this step doesn't bring much value to the reader of a BDD so could help make Scenarios more concise.